lib/PkgConfig.pm

```diff
@@ -224,13 +224,13 @@ if($ENV{PKG_CONFIG_NO_OS_CUSTOMIZATION}) {
 
 } elsif($^O eq 'freebsd') {
 
-    # TODO: FreeBSD 10's version of pkg-config does not
+    # TODO: FreeBSD 10-12's version of pkg-config does not
     # support PKG_CONFIG_DEBUG_SPEW so I can't verify
-    # the path there, but this is what it is for
-    # FreeBSD 9
+    # the path there.
     @DEFAULT_SEARCH_PATH = qw(
         /usr/local/libdata/pkgconfig
         /usr/local/lib/pkgconfig
+        /usr/libdata/pkgconfig
```

Comment:
1. Do you know when these paths were added to FreeBSD's `pkgconf`?  I am pretty certain it wasn't there in 9.  I'd prefer to only add these paths on systems where the native `pkg-config` would have added them.
 2. Are you sufre that /usr/lib/pkgconfig isn't also added to the path?

Reply:
Possilby `/usr/local/lib/pkgconfig` should be removed.  This is what is done in `Alien::pkgconf`.
https://github.com/Perl5-Alien/Alien-pkgconf/blob/master/script/probe.pl#L208-L214

Reply:
>     1. Do you know when these paths were added to FreeBSD's `pkgconf`?  I am pretty certain it wasn't there in 9.  I'd prefer to only add these paths on systems where the native `pkg-config` would have added them.

I suspect `/usr/libdata/pkgconfig` was added in [r253637](https://svnweb.freebsd.org/base?view=revision&revision=253637) from the mtree entry, but I'm not certain.

>     2. Are you sufre that /usr/lib/pkgconfig isn't also added to the path?

I don't see this directory on any FreeBSD system I have access to, nor is it specified in `/etc/mtree/BSD.usr.dist`.